applications/nrf_desktop/doc/ble_state.rst

```diff
@@ -79,10 +79,10 @@ Passkey enabled
    The callbacks can be used to achieve higher security levels.
    The passkey input is handled in the :ref:`passkey` module.
 
-.. warning::
-   The Zephyr Bluetooth peripheral by default demands the security level 3 in case the passkey authentication is enabled.
-   In case the nRF Desktop dongle is unable to achieve security level 3, it will be unable to connect with the peripheral.
-   Disable the :option:`CONFIG_BT_SMP_ENFORCE_MITM` option to allow the dongle to connect without the authentication.
+    .. warning::
+       By default, the Zephyr Bluetooth peripheral demands the security level 3 for the passkey authentication.
```

Comment:
The passkey authentication is actually used to achieve security level 3 - the passkey is the data that is used to prevent MITM (Man In The Middle) attacks. Zephyr by default requires the passkey authentication in case peripheral supports it.
The passkey is displayed by the peer (e.g. Windows 10) and it should be typed on keyboard. The dongle cannot display the password, so its not able to connect without disabling the mentioned option.